extension/jaegerremotesampling/README.md

```diff
@@ -2,7 +2,7 @@
 
 | Status                   |                       |
 | ------------------------ |-----------------------|
-| Stability                | [beta]                |
+| Stability                | [alpha]                |
```

Comment:
nit: the table alignment could be better here, now that "alpha" (5 chars) replaced "beta' (4 chars).

Reply:
if you don't want to fix it, let me know and I'll merge as is.

Reply:
fixed